packages/core/src/external/carequality/dq/process-incoming-dq.ts

```diff
@@ -28,12 +22,7 @@ export async function processIncomingRequest(
 
     return response;
   } catch (error) {
-    if (
-      error instanceof XDSUnknownPatientId ||
-      error instanceof XDSUnknownCommunity ||
-      error instanceof XDSMissingHomeCommunityId ||
-      error instanceof XDSRegistryError
-    ) {
+    if (error instanceof IHEGatewayError) {
```

Comment:
simplification since XDSUnknownPatientId etc are all instances of IHEGatewayError